recruiting/v_recruiting.prospects_report.sql

```diff
@@ -0,0 +1,22 @@
+USE gabby
+GO
+
+CREATE OR ALTER TABLE gabby.recruiting.prospects_report AS
+
+SELECT community_name
+      ,candidate_first_name
+      ,candidate_last_name
+      ,candidate_email
+      ,current_employer
+      ,community_application_status
+      ,community_application_source
+      ,candidate_source
+      ,community_application_state_added_date
+      ,community_application_state_contacted_date
+      ,community_application_state_not_interested_date
+      ,community_application_state_not_selected_date
+      ,candidate_tags_values
+      ,brand_org_field_value
+      ,candidate_id
+FROM smartrecruiters.report_prospects
+WHERE community_name LIKE '%Jami%'
```

Comment:
confirm this returns results, I think you may need to do an `IN` here

Reply:
yep it does, I get 5,395 results which is what we should expect based on the KF email. I'd rather stick with the LIKE if we can, in case they create new communities. All of ours start with "New Jami" for some reason

Reply:
oh ok I saw both "KIPP New Jersey" and "KIPP Miami" in there as well if you want to add them